assets/js/pages/AdvisoryDetails/AdvisoryDetails.jsx

```diff
@@ -5,14 +5,16 @@ import PageHeader from '@common/PageHeader';
 import ListView from '@common/ListView';
 import AdvisoryIcon from '@common/AdvisoryIcon';
 
+export const formatPackage = ({ name, version, epoch, release, arch_label }) =>
+  `${name}-${version}-${epoch}.${release}-${arch_label}`;
```

Comment:
I think you can inline this and avoid testing the implementation detail using an `{ exact: false }` in expectations. Otherwise having a more fine grained testing ends up with you doing `formatPackage(pkg) == formatPackage(pkg)`, which will always be true.

Reply:
Yeah I was thinking the same thing. I'll do that :D